NewHorizons/Utility/DebugMenu/DebugMenuPropPlacer.cs

```diff
@@ -209,6 +209,40 @@ private Vector3 DeltaSphericalPosition(GameObject prop, Vector3 deltaSpherical)
             return newSpherical;
         }
 
+        // DB_EscapePodDimension_Body/Sector_EscapePodDimension/Interactables_EscapePodDimension/InnerWarp_ToAnglerNest
+        // DB_ExitOnlyDimension_Body/Sector_ExitOnlyDimension/Interactables_ExitOnlyDimension/InnerWarp_ToExitOnly  // need to change the colors
+        // DB_HubDimension_Body/Sector_HubDimension/Interactables_HubDimension/InnerWarp_ToCluster   // need to delete the child "Signal_Harmonica"
+
+        private Vector3 SetSphericalPosition(GameObject prop, Vector3 newSpherical)
+        {
+            Transform astroObject = prop.transform.parent.parent;
+            Transform sector = prop.transform.parent;
+            Vector3 originalLocalPos = astroObject.InverseTransformPoint(prop.transform.position); // parent is the sector, this gives localPos relative to the astroobject (what the DetailBuilder asks for)
+            Vector3 sphericalPos = CoordinateUtilities.CartesianToSpherical(originalLocalPos);
+
+            if (newSpherical == sphericalPos) return sphericalPos;
+
+            Vector3 finalLocalPosition = CoordinateUtilities.SphericalToCartesian(newSpherical);
+            Vector3 finalAbsolutePosition = astroObject.TransformPoint(finalLocalPosition);
+            prop.transform.localPosition = prop.transform.parent.InverseTransformPoint(finalAbsolutePosition);
+            Logger.Log("new position: " + prop.transform.localPosition);
+
+            var onlyChangingRAndRIsNegative = false;

```

Comment:
Id like to implement this, but I could never get it to work. Maybe for a future release?

Note: implementing this would allow people to set negative values for “r”. Not a huge deal, only useful in niche situations